bears/natural_language/WriteGoodLintBear.py

```diff
@@ -0,0 +1,47 @@
+from coalib.bearlib.abstractions.Linter import linter
+
+
+@linter(executable='write-good',
+        output_format='regex',
+        output_regex=r'(?P<message>.*)\s*on\s*line\s*(?P<line>\d+)\s*at\scolumn'
+                      '\s*(?P<column>\d+)'
+        )
+class WriteGoodLintBear:
+    """
+    Lints the text files using ``write-good`` for improving proses.
+    """
+    LANGUAGES = {"Natural Language"}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Syntax', 'Formatting', 'Grammar', 'Prose', 'Writing'}
+
```

Comment:
There is neither ``Prose`` nor ``Writting`` in ``coala``, please can push another commit in ``coala`` adding them. :)